libcodechecker/server/api/report_server.py

```diff
@@ -792,6 +793,13 @@ def getRunResults(self, run_ids, limit, offset, sort_types,
                                sorted_reports.c.id == Report.id) \
                     .filter(sorted_reports.c.id.isnot(None))
 
+                # We have to order the results again because an ORDER BY in a
+                # subtable is broken by the join.
```

Comment:
Above, `ORDER BY` is used in capitals. So `JOIN` should also be, as it's a keyword.